content/en/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -29,137 +29,130 @@ further_reading:
+1. [Install and configure a Linux Datadog Agent][1].
+2. Download and install the .NET Tracer into the application environment:
+    - **Debian or Ubuntu** - Download and install the Debian package:
+      ```bash
+      curl -LO https://github.com/DataDog/dd-trace-dotnet/releases/download/v<TRACER_VERSION>/datadog-dotnet-apm_<TRACER_VERSION>_amd64.deb
+      sudo dpkg -i ./datadog-dotnet-apm_<TRACER_VERSION>_amd64.deb
+      ```
+    - **CentOS or Fedora** - Download and install the RPM package:
+      ```bash
+      curl -LO https://github.com/DataDog/dd-trace-dotnet/releases/download/v<TRACER_VERSION>/datadog-dotnet-apm-<TRACER_VERSION>-1.x86_64.rpm
+      sudo rpm -Uvh datadog-dotnet-apm-<TRACER_VERSION>-1.x86_64.rpm
+      ```
+    - **Alpine or other [musl-based distributions][2]** - Download the tar archive with the musl-linked binary:
+      ```bash
+      sudo mkdir -p /opt/datadog
+      curl -L https://github.com/DataDog/dd-trace-dotnet/releases/download/v<TRACER_VERSION>/datadog-dotnet-apm-<TRACER_VERSION>-musl.tar.gz \
+      | sudo tar xzf - -C /opt/datadog
+      ```
+    - **Other distributions** - Download the tar archive with the glibc-linked binary:
+      ```bash
+      sudo mkdir -p /opt/datadog
+      curl -L https://github.com/DataDog/dd-trace-dotnet/releases/download/v<TRACER_VERSION>/datadog-dotnet-apm-<TRACER_VERSION>.tar.gz \
+      | sudo tar xzf - -C /opt/datadog
+      ```
+3. Add the [required environment variables](#required-environment-variables).
+
+4. Run the `/opt/datadog/createLogPath.sh` script, which creates a directory for the log files and sets appropriate directory permissions. The default directory for log files is `/var/log/datadog/dotnet`.
+
+5. Create application load.
+
+6. Visit [APM Live Traces][3].
+
+
+[1]: /agent/basic_agent_usage/
 [2]: https://en.wikipedia.org/wiki/Musl
+[3]: https://app.datadoghq.com/apm/traces
 {{% /tab %}}
 
 {{< /tabs >}}
 
-### Required Environment Variables
+By default, tracing is enabled in your `datadog.yaml` file under `apm_enabled: true` and the Agent listens for trace traffic at `localhost:8126`.
+
+### Required environment variables
 
 {{< tabs >}}
 
 {{% tab "Windows" %}}
 
-If your application runs in IIS, skip the rest of this section.
-
-For Windows applications **not** running in IIS, set these two environment variables before starting your application to enable automatic instrumentation:
+#### Applications not hosted in IIS
 
-**Note:** The .NET runtime tries to load a profiler into _any_ .NET process that is started with these environment variables set. You should limit instrumentation only to the applications that need to be traced. **We do not recommend setting these environment variables globally as this causes _all_ .NET processes on the host to load the profiler.**
+To enable automatic instrumentation on For Windows applications not in IIS, you must set two environment variables before starting your application:
```

Comment:
```suggestion
To enable automatic instrumentation for Windows applications not hosted in IIS, you must set two environment variables before starting your application:
```

Reply:
heh, you can really tell I've been looking at these too long and no longer reading them. Thank you.